node/tests/swamp/swamp_tx.go

```diff
@@ -0,0 +1,45 @@
+package swamp
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-node/ipld"
+)
+
+// SubmitData submits given data in the block.
+// TODO(@Wondertan): This must be a real PFD using celestia-app, once we able to run App
+//  in the Swamp.
+func (s *Swamp) SubmitData(ctx context.Context, t *testing.T, data []byte) {
+	result, err := s.CoreClient.BroadcastTxSync(ctx, append([]byte("key="), data...))
+	require.NoError(t, err)
+	require.Zero(t, result.Code)
+}
+
+func (s *Swamp) FillBlock(ctx context.Context, t *testing.T, bsize int) {
+	data := make([]byte, bsize*ipld.ShareSize)
+	s.SubmitData(ctx, t, data)
+}
```

Comment:
Can be replaced with `swamp.FillBlocks(ctx, t, 16, 1)` if we want only 1 instead of N
```suggestion
```

Reply:
Yep. Good point.